packages/css/pagination.css

```diff
@@ -32,32 +32,32 @@
     margin-left: var(--fdsc-pagination-chevron-margin);
   }
 
-  .listitem {
+  .fds-pagination__item {
     flex: 1;
     margin-right: var(--fsdc-pagination-listitem-margin);
   }
 
-  .listitem:where(.small) {
+  .fds-pagination__item.fds-pagination--small {
     --fsdc-pagination-listitem-margin: var(--fds-spacing-2);
     --fdsc-pagination-ellipsis-width: var(--fds-sizing-10);
   }
 
-  .listitem:where(.medium) {
+  .fds-pagination__item.fds-pagination--medium {
     --fsdc-pagination-listitem-margin: var(--fds-spacing-4);
     --fdsc-pagination-ellipsis-width: var(--fds-sizing-12);
   }
 
-  .listitem:where(.large) {
+  .fds-pagination__item.fds-pagination--large {
     --fsdc-pagination-listitem-margin: var(--fds-spacing-6);
     --fdsc-pagination-ellipsis-width: var(--fds-sizing-14);
   }
 
-  .listitem:where(.compact) {
+  .fds-pagination__item.fds-pagination--compact {
```

Comment:
Suggest swapping the order for these selectors around and changing to child selector. They will then work for `fds-pagination__item` at any depth and you don't need to use `PaginationContext` in `PaginationContent`.